sa/sa.go

```diff
@@ -45,6 +46,13 @@ type SQLStorageAuthority struct {
 	// We use function types here so we can mock out this internal function in
 	// unittests.
 	countCertificatesByName certCountFunc
+
+	// failedAddCertRLTransactions is a Counter for the number of times
+	// a ratelimit update transaction failed during AddCertificate request
+	// processing. We do not fail the overall AddCertificate call when ratelimit
+	// transactions fail and so use this stat to maintain visibility into the rate
+	// this occurs.
+	failedAddCertRLTransactions prometheus.Counter
```

Comment:
This variable is pretty unwieldy and combines a shortening (AddCert) with an acronym (RL). :-D

How about rateLimitWriteErrors?

Reply:
SGTM. Thanks, renamed in fd33697